src/dnx/servicing.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#include "stdafx.h"
+#include "version.h"
+#include "utils.h"
+#include <string>
+#include <fstream>
+#include <algorithm>
+#include "TraceWriter.h"
+
+namespace
+{
+    const std::string runtime_moniker =
+#if defined(CORECLR_WIN)
+        "coreclr"
+#else
+        "clr"
+#endif
+        "-win-"
+#if defined(AMD64)
+        "x64"
+#elif defined(ARM)
+        "arm"
+#else
+        "x86"
```

Comment:
Talked to Murat - we are not adding the assert. Doesn't seem like we are going to add a new runtime or platform anytime soon and even if we did it is not guaranteed that the variables won't be defined so you won't get the assert anyways and still the moniker will be incorrect.